www/components/Function.tsx

```diff
@@ -27,16 +34,32 @@ const Container = styled(Box)`
 const FuncDocs = styled(Documentation)`
   margin-left: 1em;
 `;
+const Keyword = styled.span`
+  font-weight: bold;
+  color: rgb(249, 38, 114);
+`;
+const Fn = styled.span`
+  color: rgb(166, 226, 46);
+`;
+const Ty = Fn;
 
 export default function Function({ func }: { func: Func }) {
   const def = func.asynchronous ? "async def" : "def";
-  const ret = func.returns === null ? "" : `-> ${func.returns}`;
+  const ret =
+    func.returns === null ? (
+      <></>
+    ) : (
+      <>
+        {" -> "}
+        <Ty>{func.returns}</Ty>
+      </>
+    );
```

Comment:
is this worth factoring out? `->` vs `:` can be a parameter

Reply:
Maybe? No strong opinion either way.